Sources/TSCUtility/CMakeLists.txt

```diff
@@ -44,6 +44,7 @@ add_library(TSCUtility
   misc.swift
 )
 target_link_libraries(TSCUtility PUBLIC
+  TSCclibc
```

Comment:
```cmake
target_link_libraries(TSCUtiltity PUBLIC
  TSCBasic)
target_link_libraries(TSCUtility PRIVATE
  TSCclibc)
```

The reason to link against `TSCclibc` privately here is that users of the TSCUtility should not need to know about it now that it is `@_implementationOnly`.

Reply:
Thanks, @compnerd.  Pushed changes that implement what I think you're suggesting, which makes total sense, of course.